CRM/Case/XMLProcessor/Report.php

```diff
@@ -880,44 +880,7 @@ public static function printCaseReport() {
       }
     }
     // Retrieve ALL client relationships
-    $relClient = CRM_Contact_BAO_Relationship::getRelationship($clientID,
-      CRM_Contact_BAO_Relationship::CURRENT,
-      0, 0, 0, NULL, NULL, FALSE
-    );
-    foreach ($relClient as $r) {
-      if ($isRedact) {
-        if (!array_key_exists($r['name'], $report->_redactionStringRules)) {
-          $report->_redactionStringRules = CRM_Utils_Array::crmArrayMerge($report->_redactionStringRules,
-            array($r['name'] => 'name_' . rand(10000, 100000))
-          );
-        }
-        if (!array_key_exists($r['display_name'], $report->_redactionStringRules)) {
-          $report->_redactionStringRules[$r['display_name']] = $report->_redactionStringRules[$r['name']];
-        }
-        $r['name'] = $report->redact($r['name'], TRUE, $report->_redactionStringRules);
-
-        if (!empty($r['phone']) &&
-          !array_key_exists($r['phone'], $report->_redactionStringRules)
-        ) {
-          $report->_redactionStringRules = CRM_Utils_Array::crmArrayMerge($report->_redactionStringRules,
-            array($r['phone'] => 'phone_' . rand(10000, 100000))
-          );
-        }
-        $r['phone'] = $report->redact($r['phone'], TRUE, $report->_redactionStringRules);
-
-        if (!empty($r['email']) &&
-          !array_key_exists($r['email'], $report->_redactionStringRules)
-        ) {
-          $report->_redactionStringRules = CRM_Utils_Array::crmArrayMerge($report->_redactionStringRules,
-            array($r['email'] => 'email_' . rand(10000, 100000))
-          );
-        }
-        $r['email'] = $report->redact($r['email'], TRUE, $report->_redactionStringRules);
-      }
-      if (!array_key_exists($r['id'], $caseRelationships)) {
-        $otherRelationships[] = $r;
-      }
-    }
+    $otherRelationships = $this->getOtherRelationships($clientID, $caseID, $isRedact, $report, $caseRelationships);
```

Comment:
I think this needs to be a static function - my ide is complaining it is not in object context

Reply:
I guess it doesn't come up in running since it's never called statically. I might rethink this a little.